tests/plugins/colordialog/colordialog.js

```diff
@@ -0,0 +1,56 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: colordialog,wysiwygarea */
+
+( function() {
+	'use strict';
+
+	bender.editor = true;
+
+	bender.test( {
+		assertColor: function( inputColor, outputColor ) {
+			var editor = this.editor;
+
+			editor.once( 'dialogShow', function( evt ) {
+				var dialog = evt.data;
+				dialog.setValueOf( 'picker', 'selectedColor', inputColor );
+				dialog.getButton( 'ok' ).click();
+
+			} );
+
+			editor.getColorFromDialog( function( color ) {
+				resume( function() {
+					assert.areSame( outputColor, color );
+				} );
+			} );
+			wait();
+		},
+
+		'test colordialog add hash to colors 6 digits': function() {
```

Comment:
I think test names would be more readable like:
* `test colordialog add hash to colors - 6 digits`
* `test colordialog add hash to colors: 6 digits`
* `test colordialog add hash to colors with 6 digits color`